dev_scripts_helpers/documentation/test/test_dockerized_pandoc.py

```diff
@@ -0,0 +1,99 @@
+import os
+
+import pytest
+
+import helpers.hdocker as hdocker
+import helpers.hio as hio
+import helpers.hunit_test as hunitest
+
+
+# #############################################################################
+# Test_Pandoc_Cmd_Conversion
+# #############################################################################
+
+
+class Test_Pandoc_Cmd_Conversion(hunitest.TestCase):
+
+    def test1(self) -> None:
+        """
+        Test `convert_pandoc_cmd_to_arguments` to parse a pandoc command string
+        into a dictionary.
+        """
+        # Create a sample pandoc command string.
+        cmd = (
+            "pandoc sample.md -o output.md --data-dir data "
+            "--template default --extract-media media -- --verbose --extra"
+        )
+        # Call function to test.
+        result = hdocker.convert_pandoc_cmd_to_arguments(cmd)
+        expected = {
+            "input": "sample.md",
+            "output": "output.md",
+            "in_dir_params": {
+                "data-dir": "data",
+                "template": "default",
+                "extract-media": "media",
+            },
+            "cmd_opts": ["--verbose", "--extra"],
+        }
+        # Check output.
+        self.assertEqual(result, expected)
+
+    def test2(self) -> None:
+        """
+        Test `convert_pandoc_arguments_to_cmd` to build a command string from a
+        dictionary of parameters.
+        """
+        # Prepare inputs.
+        params = {
+            "input": "sample.md",
+            "output": "output.md",
+            "in_dir_params": {
+                "data-dir": "data",
+                "template": "default",
+                "extract-media": "media",
+            },
+            "cmd_opts": ["--verbose", "--extra"],
+        }
+        # Call function to test.
+        cmd = hdocker.convert_pandoc_arguments_to_cmd(params)
+        expected_cmd = (
+            "sample.md --output output.md --data-dir data --template default "
+            "--extract-media media --verbose --extra"
+        )
+        # Check output.
+        self.assertEqual(cmd, expected_cmd)
+
+
+# #############################################################################
+# Test_run_dockerized_pandoc
+# #############################################################################
+
+
+class Test_run_dockerized_pandoc(hunitest.TestCase):
+
+    @pytest.mark.timeout(10)
```

Comment:
Can you mark it as superslow? There is a tag.

You can grep the documentation to see why / how.